### PR TITLE
Release/4.0.0

### PR DESCRIPTION
**Pull request by piyalbasu**

### CODE REVIEW

File: extension/src/popup/components/mnemonicPhrase/ConfirmMnemonicPhrase/index.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ export const ConfirmMnemonicPhrase = ({
         </OnboardingHeader>
         <div className="ConfirmMnemonicPhrase__content">
           <p>{t("Please select each word in the same order you have")}</p>
-          <p>{t("them noted to confirm you got them right")}</p>
+          <p>{t("them noted to confirm you go them right")}</p>
```
Comment (aristidesstaffieri) [inline extension/src/popup/components/mnemonicPhrase/ConfirmMnemonicPhrase/index.tsx:77]: Probably not a big deal but w're undoing this typo fix here.

File: @stellar/freighter-api/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stellar/freighter-api",
-  "version": "1.4.1",
```
Comment (aristidesstaffieri) [inline @stellar/freighter-api/package.json:3]: Do we want to keep this version on the current one?

File: extension/src/popup/components/sendPayment/SendAmount/index.tsx [hunk 3]
```diff
@@ -143,39 +141,32 @@ export const SendAmount = ({
...
             accountBalances.balances[selectedAsset].total.toFixed(),
           );
-
           availBalance = currentBal
-            .minus(minBalance)
-            .minus(new BigNumber(Number(recommendedFee)));
-
-          if (availBalance.lt(minBalance)) {
-            return "0";
-          }
```
Comment (aristidesstaffieri) [inline extension/src/popup/components/sendPayment/SendAmount/index.tsx:169]: I believe we want to stick with the previous version here that matches master branch, it fixes a display bug for balances.

File: @stellar/freighter-api/package.json (hunk 2 above)
Comment (piyalbasu) [reply @stellar/freighter-api/package.json:3]: Oh yup, noticed this, too. This was actually a mistake to begin with. The current version on NPM is 1.4.0

**Review by aristidesstaffieri — APPROVED** [review]
lgtm 🤞

**Pull request merged**